apps/web/composables/useAggregatedCountries/useAggregatedCountries.ts

```diff
@@ -0,0 +1,54 @@
+import { type AggregatedCountries } from '@plentymarkets/shop-api';
+import { type UseAggregatedCountriesReturn, UseAggregatedCountriesState, type GetAggregatedCountries } from './types';
+
+/**
+ * @description Composable for getting `AggregatedCountries`:
+ * - Active shipping countries
+ * - EU geo-regulated countries
+ *
+ * @example
+ * ``` ts
+ * const {
+ *  data,
+ *  loading,
+ *  useGeoRegulatedCountries,
+ *  getAggregatedCountries,
+ *  getDefaultCountries,
+ *  getGeoRegulatedCountries,
+ * } = useAggregatedCountries();
+ * ```
+ */
+export const useAggregatedCountries: UseAggregatedCountriesReturn = () => {
+  const state = useState<UseAggregatedCountriesState>('useAggregatedCountries', () => ({
+    data: {} as AggregatedCountries,
```

Comment:
```suggestion
    data: {
      default: [],
      geoRegulated: []
    } as AggregatedCountries,
```

Reply:
if we change it so that the state data uses the default and geoRegulated with the empty array as a default, we can avoid the length checks right?

Reply:
I think it would be more in line with the rest of the app if we use the destructed data like { geoRegulated: geoRegulatedCountries } = useAggregatedCountries(); instead of `getGeoRegulatedCountries`